### PR TITLE
Add fixes for documentations

### PR DESCRIPTION
**Pull request by AathmanT**

### CODE REVIEW

File: openai.audio/client.bal [hunk 1]
```diff
@@ -60,6 +60,7 @@ public isolated client class Client {
     # Transcribes audio into the input language.
     #
     # + return - OK 
+    @display {label: "Create Transcription"}
```
Comment (NipunaRanasinghe) [inline openai.audio/client.bal:63]: Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

File: openai.audio/client.bal [hunk 2]
```diff
@@ -71,6 +72,7 @@ public isolated client class Client {
     # Translates audio into into English.
     #
     # + return - OK 
+    @display {label: "Create Translation"}
```
Comment (NipunaRanasinghe) [inline openai.audio/client.bal:75]: Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

File: openai.finetunes/client.bal [hunk 3]
```diff
@@ -80,6 +82,7 @@ public isolated client class Client {
     #
     # + file_id - The ID of the file to use for this request 
     # + return - OK 
+    @display {label: "Retrieve File"}
```
Comment (NipunaRanasinghe) [inline openai.finetunes/client.bal:85]: Shall we use "Get File", "Get file content", etc for all the getter methods to be consistent with the other naming for other method types?
```suggestion
    @display {label: "Get File"}
```

Comment (NipunaRanasinghe) [reply openai.finetunes/client.bal:85]: Please consider this for the other relevant methods as well

File: openai.chat/client.bal [hunk 4]
```diff
@@ -59,6 +59,7 @@ public isolated client class Client {
     # Creates a completion for the chat message
     #
     # + return - OK 
+    @display {label: "Create Chat Completion"}
```
Comment (NipunaRanasinghe) [inline openai.chat/client.bal:62]: Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

File: openai.text/client.bal [hunk 5]
```diff
@@ -59,6 +59,7 @@ public isolated client class Client {
     # Creates a completion for the provided prompt and parameters
     #
     # + return - OK 
+    @display {label: "Create Completion"}
```
Comment (NipunaRanasinghe) [inline openai.text/client.bal:62]: Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

File: openai.audio/client.bal (hunk 1 above)
Comment (AathmanT) [reply openai.audio/client.bal:63]: I actually did those too but haven't pushed them. Please check now.

File: openai.audio/client.bal (hunk 2 above)
Comment (AathmanT) [reply openai.audio/client.bal:75]: I actually did those too but haven't pushed them. Please check now.

File: openai.chat/client.bal (hunk 4 above)
Comment (AathmanT) [reply openai.chat/client.bal:62]: I actually did those too but haven't pushed them. Please check now.

File: openai.text/client.bal (hunk 5 above)
Comment (AathmanT) [reply openai.text/client.bal:62]: I actually did those too but haven't pushed them. Please check now.

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**